packages/hoprd/Dockerfile

```diff
@@ -45,17 +45,17 @@ COPY .nvmrc package.json ./
 COPY scripts/toolchain/install-toolchain.sh ./scripts/toolchain
 
 RUN apt-get update \
- && apt-get install -y \
-      bash curl jq tini ca-certificates tar xz-utils \
- && rm -rf /var/lib/apt/lists/* \
- && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
- # Install Node.js
- && ./scripts/toolchain/install-toolchain.sh --runtime-only \
- # Remove unused files and config
- && rm -R scripts .nvmrc \
- # create directory which is later used for the database, so that it inherits
- # permissions when mapped to a volume
- && mkdir -p hoprd-db
+    && apt-get install -y \
```

Comment:
Please don't change the whitespace here. We can agree on a different style separately.

Reply:
oh, that was done automatically ..